tools/smoother_benchmarking/README.md

```diff
@@ -0,0 +1,121 @@
+# Planners Smoothing Benchmark
+
+This experiment runs a set with randomly generated goals for objective benchmarking.
+
+Bechmarking scripts require the following python packages to be installed:
+
+```
+pip install transforms3d
+pip install seaborn
+pip install tabulate
+```
+
+To perform the benchmarking, the following changes should be made for planner and smoother servers:
+
+```
+diff --git a/nav2_planner/src/planner_server.cpp b/nav2_planner/src/planner_server.cpp
+index c7a90bcb..510bb6ea 100644
+--- a/nav2_planner/src/planner_server.cpp
++++ b/nav2_planner/src/planner_server.cpp
+@@ -285,7 +285,7 @@ bool PlannerServer::getStartPose(
+   typename std::shared_ptr<const typename T::Goal> goal,
+   geometry_msgs::msg::PoseStamped & start)
+ {
+-  if (goal->use_start) {
```

Comment:
I don't understand this diff, you can just set `use_start = True` in your `getPlan` request and it'll do this without modifying the source code

Reply:
Yep. Missed it. Set `use_start = True` for `getPath` [instead](https://github.com/ros-planning/navigation2/pull/3236/files#diff-65e8732e1465516bc6d72d450f91fb172336c4e37ece025895c3e197ef6432a6R36).